src/android/MixpanelPlugin.java

```diff
@@ -229,7 +229,7 @@ private boolean handlePeopleSetOnce(JSONArray args, final CallbackContext cbCtx)
 
     private boolean handlePeopleSetPushId(JSONArray args, final CallbackContext cbCtx) {
         String pushId = args.optString(0);
-        mixpanel.getPeople().setPushRegistrationId(pushId);
+        mixpanel.getPeople().setPushRegistrationId('"'+pushId+'"');
```

Comment:
You're absolutely right. I'll Do that tomorrow and send a new pull-request.
/Simon

Skickat från min iPhone

> 4 nov. 2015 kl. 19:53 skrev Sam Zilverberg <notifications@github.com>:
> 
> In src/android/MixpanelPlugin.java:
> 
> > @@ -229,7 +229,7 @@ private boolean handlePeopleSetOnce(JSONArray args, final CallbackContext cbCtx)
> >  
> >      private boolean handlePeopleSetPushId(JSONArray args, final CallbackContext cbCtx) {
> >          String pushId = args.optString(0);
> > -        mixpanel.getPeople().setPushRegistrationId(pushId);
> > +        mixpanel.getPeople().setPushRegistrationId('"'+pushId+'"');
> whaaaat why? :)
> can't see any other examples floating around with that kind of fix.
> 
> Maybe its related to this issue on mixpanel-android github:
> mixpanel/mixpanel-android#227
> this was fixed in 4.7.0
> 
> I'd rather update to 4.7.0 and try it again, if you got time soon you can try replacing the lib in the plugin dir and perhaps reference in plugin.xml.
> 
> i'll try to check into it on the weekend if you dont get to it.
> 
> —
> Reply to this email directly or view it on GitHub.
>